css-two-column-layout/styles.css

```diff
@@ -0,0 +1,76 @@
+* {
+  font-family: "Gill Sans", sans-serif;
+  box-sizing: border-box;
+}
+
+.container {
+  justify-content: center;
+  max-width: 1000px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 70px;
+}
+
+.head {
+  text-align: center;
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
+
+.p-head {
+  font-size: 13px;
+  width: 75%;
+  margin-left: 100px;
+  letter-spacing: 1px;
+}
+
+.row {
+  display: flex;
+}
+
+.column-fourth {
+  display: flex;
+
+  /* width: calc(100% / 4); */
+
+  width: 75%;
```

Comment:
I would try a 50% width, since each column is supposed to be one half of the row

Reply:
![image](https://user-images.githubusercontent.com/22688744/151598923-61c4aa1d-752c-454d-8651-61b43abeef60.png)